standalone/src/main/java/io/pravega/local/SingleNodeConfig.java

```diff
@@ -20,6 +20,12 @@
     public final static Property<Integer> SEGMENTSTORE_PORT = Property.named("segmentstorePort", 6000);
     public final static Property<Integer> CONTROLLER_PORT = Property.named("controllerPort", 9090);
     public final static Property<Integer> REST_SERVER_PORT = Property.named("restServerPort", 9091);
+    public final static Property<String> CERT_FILE = Property.named("certFile", "../config/cert.pem");
+    public final static Property<String> KEY_FILE = Property.named("keyFile", "../config/key.pem");
+    public final static Property<String> PASSWD_FILE = Property.named("passwdFile", "../config/passwd");
+    public final static Property<String> USER_NAME = Property.named("userName", "admin");
+    public final static Property<String> PASSWD = Property.named("passwd", "1111_aaaa");
+
```

Comment:
Why do we have these default values?   If you want a 'secure by default' configuration, you need to generate a cert and generate a passwd file.  If not, these defaults complicate life without a real benefit.

Reply:
`secure by default` is chosen by `enableAuth` and `enableTls` settings. These defaults are here so that users can just set one value to toggle between secure and insecure options.
The default is insecure.